lib/cc/engine_registry.rb

```diff
@@ -27,7 +27,7 @@ def each
 
     def fetch_engine_details(engine, development: false)
       if development
-        EngineDetails.new("codeclimate/codeclimate-#{engine.name}", nil, "")
+        EngineDetails.new("codeclimate/codeclimate-#{engine.name}", nil, "", nil)
```

Comment:
I don't think you have to explicitly pass in `nil` here. Any missing arguments will default to a `nil` value:

```ruby
EngineDetails.new("codeclimate/codeclimate-#{engine.name}", nil, "")
```